front/lib/actions/registry.ts

```diff
@@ -193,6 +193,22 @@ export const DustProdActionRegistry: {
       },
     },
   },
+  "extract-events": {
+    app: {
+      workspaceId: PRODUCTION_DUST_APPS_WORKSPACE_ID,
+      appId: "edf70ecf98",
+      appHash:
+        "093b3466de48a8c6a70b12aaf88404219038a01462341d3b628819d39fa5ca07",
```

Comment:
Looking at the app I think you can template the event string in the instructions of the MODEL block so that the model does not see [[name_of_event]] but [[product_idea]] or [[idea]].